### DISCUSSION TITLE
Contrato programador MAPos

### DISCUSSION THREAD

**Discussion by romerorigon** [RamonSilva20/mapos#2480] [General]
Olá gostaria de saber se há algum programador para efetuar algumas alterações no MapOS.
Exemplo:
2 locais de estoque
Cadastro de equipamentos (anteriormente existia)
Assinatura nas OSs

**Comment by WesMacedo** [comment]
Possso estar fazendo essas alterações para você. Chama no WhatsApp: (88) 9 9212-8803
